src/Microsoft.DotNet.Build.Tasks/PropertyDefinitions.cs

```diff
@@ -86,12 +86,12 @@ public PropertyDefinitions()
 
         private readonly ContentPropertyDefinition _assembly = new ContentPropertyDefinition
         {
-            FileExtensions = { ".dll" }
+            FileExtensions = { ".dll", "_._" }
```

Comment:
_._ is a placeholder file.  It's a convention used in nuget to indicate that the particular pivot is supported but no assets apply.